tests/phpunit/CRM/Contribute/BAO/ContributionTest.php

```diff
@@ -37,6 +37,16 @@ public function tearDown(): void {
     parent::tearDown();
   }
 
+  public function testAPiv4Validates() {
+    $this->expectExceptionMessage("fail");
+    Contribution::create()->setValues([
+      'total_amount' => 5,
+      'contact_id' => $this->individualCreate(),
+      'financial_type_id' => 1,
+      'contribution_status_id' => 'I made up some rubbish'
```

Comment:
@eileenmcnaughton so the Rabbit didn't eat this rubbish?

Reply:
@seamuslee001 no - the rabbit has fine taste in rubbish - it prefers rubbish that is still in use - e.g  - our walls